File: 백준/Bronze/14568. 2017 연세대학교 프로그래밍 경시대회/2017 연세대학교 프로그래밍 경시대회.py
import sys

input = sys.stdin.readline

n = int(input())

ans = 0

for i in range(0,n+1):
    for j in range(0,n+1):
        for k in range(0,n+1):
            if i + j + k == n:
                if k >= j + 2:
                    if i % 2 == 0:
                        if i > 0 and j > 0 and k > 0:
                            ans += 1
print(ans)

File: 백준/Bronze/14568. 2017 연세대학교 프로그래밍 경시대회/2017 연세대학교 프로그래밍 경시대회.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());
        int ans = 0;

        for (int i = 0; i < n + 1; i++) {
            for (int j = 0; j < n + 1; j++) {
                for (int k = 0; k < n + 1; k++) {
                    if (i + j + k == n) {
                        if (k >= j + 2) {
                            if (i > 0 && j > 0 && k > 0) {
                                if (i % 2 == 0) {
                                    ans++;
                                }
                            }
                        }
                    }
                }
            }
        }

        System.out.println(ans);

    }
}